src/commands/synthetics/run-tests-command.ts

```diff
@@ -158,6 +154,13 @@ export class RunTestsCommand extends Command {
       return 1
     }
 
+    const reporters: Reporter[] = [new DefaultReporter(this)]
+    this.reporter = getReporter(reporters)
+
+    if (this.config.jUnitReport) {
+      reporters.push(new JUnitReporter(this))
```

Comment:
I think the `JUnitReporter` expects there to be a top-level `jUnitReport` available. We should change this behavior to rely on `this.config.jUnitReporter` instead